requirements.txt

```diff
@@ -59,11 +59,11 @@ django-debug-toolbar==1.3.2
 django-waffle==0.10.1
 django-jenkins==0.17.0
 django-smoketest==0.3.0
-django-extensions==1.1.0
+django-extensions==1.5.2
 django-stagingcontext==0.1.0
 django-impersonate==0.9.1
 django-registration-redux==1.1
```

Comment:
django-registration-redux should be updated to 1.2, according to http://wiki.ccnmtl.columbia.edu/index.php/Django_1.8_Upgrade

Otherwise I've tested this branch and haven't seen any problems.